### ISSUE TITLE
dev-lang/rust-1.28.0 uses wrong stage-0 on armv7

### ISSUE THREAD

**Issue by stefson** [smaeul/portage-overlay#14]
here is the full build log: [rust-1.28.0-failed.log.gz](https://github.com/smaeul/portage-overlay/files/2498340/rust-1.28.0-failed.log.gz)

and emerge --info: [emerge--info.gz](https://github.com/smaeul/portage-overlay/files/2498341/emerge--info.gz)

Honestly I don't understand why this happens?

**Comment by smaeul** [comment]
Yes, the correct way to handle this was to create real targets using the Gentoo triples, instead of doing string mangling. Unfortunately, this doesn't quite solve the other issue on ARM, which is that there are three different CHOSTs for the same ARCH. I think adding all of them to SRC_URI should be enough.
